### ISSUE TITLE
IPv6 Kubernetes API server address parsed as invalid URI

### ISSUE THREAD

**Issue by abroglesc** [falcosecurity/falco#2975]
**Describe the bug**
When attempting to run falco on a kubernetes cluster with an IPv6 based networking scheme and a `KUBERNETES_SERVICE_HOST` variable that evaluates to: `https://fdf3:84f9:f47b::1` I receive the following error from the libs during startup:
```
[libs]: Error fetching K8s data: Invalid URI: [https://fdf3:84f9:f47b::1]
```

**How to reproduce it**
Have a kubernetes cluster with IPv6 based networking or an easier way to reproduce is pass `-k "https://fdf3:84f9:f47b::1"` to falco directly during startup. Passing this directly should trigger this invalid URI codepath.

**Expected behaviour**
This is in fact a valid IPv6 URI and I would expect Falco to startup properly and contact the kubernetes API server at that address.

**Environment**
- Falco version: 0.34.1
- driver name: falco                                                                                                                     
- target identifier: amazonlinux2                                                                                                        
- kernel release: 5.10.130-118.517.amzn2.aarch64                                                                                         
- kernel version: 1

- Installation method:
Kubernetes

**Additional context**
This appears to be in this codepath: https://github.com/falcosecurity/libs/blob/master/userspace/libsinsp/sinsp.cpp#L2488

And ultimately comes down to Falco's custom URI parser that is based off of an nginx implementation: https://github.com/falcosecurity/libs/blob/master/userspace/libsinsp/uri_parser.c#L584-L614

**Comment by Andreagit97** [comment]
this should be solved by the new `k8smeta` plugin that we will release with Falco 0.37.0

**Comment by Andreagit97** [comment]
The new `k8smeta` plugin should solve this issue, give it a try with our dedicated helm chart and Falco 0.36.2 https://github.com/falcosecurity/falco/issues/2973#issuecomment-1876880587

**Comment by Andreagit97** [comment]
This should be solved by [Falco 0.37.0](https://github.com/falcosecurity/falco/releases/tag/0.37.0)! Feel free to reopen if this is still an issue

**Comment by nickpetrovic** [comment]
Recently ran into this. I can confirm Falco works in an IPv6 network, but I had to set the web server's listen address in the helm chart like so:

```yaml
falco:
  # ...
  webserver:
    enabled: true
    listen_address: "::"
```
